README.md

```diff
@@ -105,9 +107,6 @@ If your changes do not break the current environment, you can submit a separate
 
 Otherwise, you will need to speak to an admin who can force merge your full changes after testing locally.
 
-### Building for Android
-Note there is an article written for how to build Holochain for Android, read it [here](doc/holochain_101/src/holochain_across_platforms.md).
-
```

Comment:
Why remove this?

Reply:
The reason, at least for me, is because it feels a little arbitrary to put this in the readme, but not other info, that's in the book. We already link to the book, and 'building for android' is in the chapter structure, so I don't know exactly why we'd include it here